pypred/compact.py

```diff
@@ -32,10 +32,10 @@ def node_name(node):
     elif cls_name == "NegateOperator":
         return (cls_name, node_name(node.left))
 
-    elif cls_name in ("CompareOperator", "LogicalOperator"):
+    elif cls_name in ("CompareOperator", "LogicalOperator", "ContainsOperator"):
```

Comment:
IIUC, this is the approach pypred uses for polymorphism? I.e `CompareOperator` is the class, and the type is provided by `node.type` so that the class can know to compare greater than equal, less than, etc.? Hence we do this because we need to know the type of contains (regular vs contains any vs contains anyof)

Reply:
Exactly!